examples/hwc_example.cpp

```diff
@@ -81,47 +81,51 @@ int main(int argc, char* argv[])
     videoPipe = dpipe_lookup("video-0"))
     */
 
-    //create hwc receiver
-    VirtualHwcReceiver* receiver = new VirtualHwcReceiver(cfg, cmd_handler);
-    IOResult ior = receiver->start();
-    int index = 0;
-    while (!std::get<0>(ior)) {
-        index ++;
-
-        if (index % 300 == 0) {
-            printf("tag: index(%d)\n", index);
+    try {
+        //create hwc receiver
+        VirtualHwcReceiver* receiver = new VirtualHwcReceiver(cfg, cmd_handler);
```

Comment:
I suggest to use smart pointer to simplify code (this will allow to drop explicit `delete` calls):
```
std::unique_ptr<VirtualHwcReceiver> = std:::make_unique<VirtualHwcReceiver>(cfg, cmd_handler);
```

Reply:
In actual, my change just added try-catch to handle exception, Coverity tool complains that none will capture exception raised in the constructor of VirtualHwCReceiver.